cluster/ci/config/prow/plugins.yaml

```diff
@@ -1038,6 +1038,10 @@ config_updater:
       name: job-config-4.3
     ci-operator/jobs/**/*openshift-4.3*submits.yaml:
       name: job-config-4.3
+    ci-operator/jobs/openshift/kubernetes/*.yaml:
+      name: ci-operator-misc-configs
```

Comment:
job-config-misc

Reply:
yeah, just figured it as well, that what happens when you copy paste in a hurry :)